editor.planx.uk/src/pages/FlowEditor/lib/store/preview.ts

```diff
@@ -33,24 +34,31 @@ import type { SharedStore } from "./shared";
 
 const SUPPORTED_DECISION_TYPES = [TYPES.Checklist, TYPES.Question];
 let memoizedPreviousCardId: string | undefined = undefined;
-let memoizedBreadcrumb: Store.breadcrumbs | undefined = undefined;
+let memoizedBreadcrumb: Store.Breadcrumbs | undefined = undefined;
+
+export interface Response {
+  question: Node & { id: NodeId };
+  selections: Array<Node & { id: NodeId }>;
+  hidden: boolean;
+}
```

Comment:
This is pulled out here to reuse existing types, rather than redefine them (as seen in `editor.planx.uk/src/types.ts`)